### PR TITLE
strimzi impl

### LINKED ISSUE #35: strimzi

### PR DESCRIPTION
**Pull request by eyakauleva**
close #35

### CODE REVIEW

File: src/infra/app-deployment.yaml [hunk 1]
```diff
@@ -11,8 +11,8 @@ spec:
     metadata:
       labels:
         app: user
-        sidecar.istio.io/inject: "true"
-        auth: jwt
+        #sidecar.istio.io/inject: "true"
```
Comment (h1alexbel) [inline src/infra/app-deployment.yaml:14]: @eyakauleva why do you remove sidecar injection?

File: src/infra/istio/istio-setup.sh [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
+kubectl apply -f auth-policy.yaml
+
+minikube tunnel
+
+
+#-----
 
-minikube tunnel
\ No newline at end of file
+#kubectl delete ingress app-ingress
```
Comment (h1alexbel) [inline src/infra/istio/istio-setup.sh:16]: @eyakauleva comments should be removed

File: src/infra/app-deployment.yaml (hunk 1 above)
Comment (eyakauleva) [reply src/infra/app-deployment.yaml:14]: @h1alexbel cause my laptop can't handle both istio and strimzi

File: src/infra/istio/istio-setup.sh (hunk 2 above)
Comment (eyakauleva) [reply src/infra/istio/istio-setup.sh]: fixed 18805b9

File: src/infra/app-deployment.yaml (hunk 1 above)
Comment (h1alexbel) [reply src/infra/app-deployment.yaml:14]: @eyakauleva it's sad(

Comment (eyakauleva) [reply src/infra/app-deployment.yaml:14]: @h1alexbel It seems you forgot to approve the PR

**Pull request merged**